lib/dispatch-rider/logging/base_formatter.rb

```diff
@@ -0,0 +1,30 @@
+module DispatchRider
+  module Logging
+    class BaseFormatter
+      def format_error_handler_fail(message, exception)
```

Comment:
Unused method argument - `message`. If it's necessary, use `_` or `_message` as an argument name to indicate that it won't be used. You can also write as `format_error_handler_fail(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `exception`. If it's necessary, use `_` or `_exception` as an argument name to indicate that it won't be used. You can also write as `format_error_handler_fail(*)` if you want the method to accept any arguments but don't care about them.